examples/megaclc/megaclc.cpp

```diff
@@ -3239,6 +3240,45 @@ void exec_getmegaachievements(ac::ACState& s)
     g_megaApi->getMegaAchievements(listener);
 }
 
+void exec_setCameraUploadsFolder(ac::ACState& s)
+{
+    std::unique_ptr<m::MegaNode> srcnode(g_megaApi->getNodeByPath(s.words[1].s.c_str()));
+    
+    if (!srcnode)
+    {
+        cout << "folder not found";
```

Comment:
Should we use a `conlock(cout)` here?
```suggestion
        conlock(cout) << "Folder not found: " << s.words[1].s << endl;
```

Reply:
good point, fixed :-)